sync-core/src/main/java/com/cloudant/sync/query/QuerySqlTranslator.java

```diff
@@ -468,6 +498,27 @@ protected static SqlParts whereSqlForAndClause(List<Object> clause, String index
         return SqlParts.partsForSql(where, parameterArray);
     }
 
+    private static String buildWhereClauseArgument(List<Object> values,
```

Comment:
If we're supporting this, we might want to consider what happens if we use too many placeholders (SQLite has a limit per query).

See (9) at https://www.sqlite.org/limits.html. Previously we've assumed 500 max to be on the safe side.